tools/tune/eq/example_iir_eq.m

```diff
@@ -192,6 +194,33 @@ function example_iir_eq()
 bp = eq_iir_blob_pack(bm);
 eq_alsactl_write(alsa_fn, bp);
 
+%% --------------------------------------------
+%% Example 7: 50 Hz high-pass with gain, 16 kHz
+%% --------------------------------------------
+tplg_fn = '../../topology/m4/eq_iir_coef_highpass_50hz_20db_16khz.m4';
+comment = '50 Hz second order high-pass and +20 dB gain';
+
+%% Design IIR high-pass
+eq_hp = hp_iir_eq(16e3);
```

Comment:
is this is just using a biquad to do the dcblocking effect? Just wondering about porting over our first order solution as well so we can port our tuning configs straight over when the time comes.

Reply:
Yes it's the simple N biquads in series filter. It's now optimized with intrinsics for HiFi3 so there's not much overhead. This is just a single biquad. A 1st order version would work here as well. I've configured this to provide also the 20 dB gain since today we cannot define in topology the initial volume gain to be other than firmware default 0 dB.

Reply:
If we will able set the initial gain for volume instances in topology I'd revert the gain of high-pass to 0 dB. With this topology the user is not able lower the gain if the capture would clip under high sound pressure levels. But since the devices that use this topology are notebooks I'd assume it's not an issue in practice.

The initial volume passing to firmware from topology would need an ABI bump. It's one idea to address the issue with default gains.

Also the volume scale could be defined to be e.g. +20 .. +50 dB then volume comp would use the lowest gain in scale as default. But then gain could not be adjusted to any lower than +20 dB. So This is what I proposed then as full rationale explained!